.ci/Jenkinsfile

```
@@ -209,6 +202,85 @@ def withKubernetes(Closure body) {
     }
 }
 
+def prepareStack() {
+  def stackArgs = '-v'
+  def manifest = readYaml(file: "manifest.yml")
+
+  if (manifest != null && manifest["conditions"] != null) {
+    def kibanaVersionCondition = manifest["conditions"]["kibana.version"]
+    if (kibanaVersionCondition != null) {
+      def version = findStackVersion(kibanaVersionCondition)
```

As soon as https://github.com/elastic/apm-pipeline-library/pull/1248 gets merged and released then:

```suggestion
      def version = findOldestSupportedVersion(versionCondition: kibanaVersionCondition)
```